# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: target/archivista/pkg_linux26.patch
# Copyright (C) 2006 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Disable SMP by default, it can be explicitly turned on again by adding maxcpu=
...

Also default to a quiet kernel.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.26/init/main.c.vanilla	2008-07-16 20:36:33.000000000 +0200
+++ linux-2.6.26/init/main.c	2008-07-16 20:36:51.000000000 +0200
@@ -130,7 +130,7 @@
 
 #ifdef CONFIG_SMP
 /* Setup configured maximum number of CPUs to activate */
-unsigned int __initdata setup_max_cpus = NR_CPUS;
+unsigned int __initdata setup_max_cpus = 1;
 
 /*
  * Setup routine for controlling SMP activation
--- ./kernel/printk.c.orig	2006-10-15 09:56:14.000000000 +0000
+++ ./kernel/printk.c	2006-10-15 10:00:08.000000000 +0000
@@ -41,7 +41,7 @@
 
 /* We show everything that is MORE important than this.. */
 #define MINIMUM_CONSOLE_LOGLEVEL 1 /* Minimum loglevel we let people use */
-#define DEFAULT_CONSOLE_LOGLEVEL 7 /* anything MORE serious than KERN_DEBUG */
+#define DEFAULT_CONSOLE_LOGLEVEL 4 /* like "quiet", just built-in */
 
 DECLARE_WAIT_QUEUE_HEAD(log_wait);
 
